cfgov/jinja2/v1/index.html

```diff
@@ -403,7 +223,7 @@ <h3 class="h1 m-card_heading">98% of consumers</h3>
                 <div class="content-l">
                     <section class="content-l_col
                                     content-l_col-1-3">
-                        <div class="">
+                        <div>
```

Comment:
Yes and no, these should eventually become related posts, but those had some layout styles that need to be pulled out before they can be used here.